File: objcetive-c/MRC/MRC/ViewController.m
//
//  ViewController.m
//  MRC
//
//  Created by ilya on 28.03.2019.
//  Copyright © 2019 ilya. All rights reserved.
//

#import "ViewController.h"
#import "MyObject3.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    MyObject3 *obj3 = [MyObject3 new];
    [obj3 test];
}

- (void)dealloc {
    
    [super dealloc];
}

@end

File: swift/Groundworks/Challenges/Challenges/Supporting Files/ViewController.swift
//
//  ViewController.swift
//  Challenges
//
//  Created by ilya on 22.04.2019.
//  Copyright © 2019 ilya. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        let matches = PatternMathing()
//        matches.partialMatches()
        matches.simpleFor()
        matches.matchingOptionals()
        
        let filesTask = TaskOnFiles()
        filesTask.challenge2(log: "Hello", to: "challenge2")
        
        filesTask.challenge3()
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destination.
        // Pass the selected object to the new view controller.
    }
    */

}
